code/style.css

```diff
@@ -1,5 +1,35 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
+
 body {
```

Comment:
To make the image go aaaall the way to the edges, you can add `margin: 0;` to get rid of the body's default margin!